website_chd_product_configurator/templates/templates.xml

```diff
@@ -0,0 +1,369 @@
+                                                            <li t-att-value="curr_accessory.id" >
+                                                            <div class="row" style="border-bottom: 0.5px solid;padding-top: 2px;padding-bottom: 2px;">
+                                                                <div class="col-xs-7">
+                                                                    <label class="checkbox-inline">
+                                                                        <p> <t t-esc="curr_accessory.name"/></p>
+                                                                    </label>
+                                                                </div>
+                                                                <div class="col-xs-5" style="border-left: 0.5px solid;">
+                                                                    <label id="small_listquantity" for="quantity_acc">Quantity</label>
+                                                                    <input t-attf-name="qtyaccessoryid_#{str(curr_accessory.id)}_#{str(curr_accessory.name)}" type="number"  min="0" t-att-maxlength="3" t-att-size="4" />
+                                                                </div>
+                                                            </div>
+                                                            </li>
+                                                  </t>
+                                        </ul>
+                               </div>
+                           </div>
+                           <div class="form-group">
+                               <div class="col-xs-4">
+
+                               </div>
+                               <div class="col-xs-2">
+                                    <button type="submit" class="btn btn-primary btn-lg pull-center" >See Quotations</button>
+                                </div>
+                            </div>
+                    <input type="hidden" name="product_id" t-att-value="curr_product_id.id"/>
+                    <input type="hidden" name="product_id_chd_size_type" t-att-value="curr_product_id.chd_size_type"/>
+                 </div>
+                     <div  class="col-sm-3">
+                            <div t-if="curr_product_id.chd_configurator_has_image" id="imagearea">
+                                  <input class="btn btn-default btn-lg pull-left" type="file" name="pic" accept="image/*" required="true"/>
+                            </div>
+                            <div class="row"> <p>
+                            	<br/>
+                            </p></div>
+
+                            <div id="feedback">
+			                	  <div id="error_msg" t-if="errormsg" class="row">
+										<h5 style="color:red;"> No result found for the values that you entered. We would be happy to give you a custom quote.
+                        Please call 010-7856766</h5>
```

Comment:
Get that from some company_id in of the browse records you have in this template